.travis.yml

```
@@ -1,26 +1,31 @@
 language: ruby
+os: linux
+dist: xenial
 cache: bundler
 rvm:
 - &latest_ruby 2.7
 - 2.5
-matrix:
+env:
+  - JEKYLL_VERSION="~> 3.7.0"
+  - JEKYLL_VERSION="~> 3.8.0"
+  - JEKYLL_VERSION="~> 4.0"
+jobs:
   include:
   # GitHub Pages
-  - rvm: 2.5.3
-    env:
-    - JEKYLL_VERSION="~> 3.8.5"
-    - GITHUB_PAGES=1 # Only set on one build in matrix
+  - rvm: 2.5
```

```suggestion
  - rvm: 2.7
```
https://pages.github.com/versions/